quiz.html

```diff
@@ -8,16 +8,16 @@
 </head>
 <body>
   <header>
-    <h1>Question #</h1>
+    <h1>Question #<span id="question-number"></span></h1>
   </header>
 
   <div class="container">
-    <div id="question-container">
+    <div id="question-container"><span id="question-content"></span>
       <div id="answer-buttons">
-        <button class="btn">Answer 1</button>
-        <button class="btn">Answer 2</button>
-        <button class="btn">Answer 3</button>
-        <button class="btn">Answer 4</button>
+        <button class="btn" id="a1">Answer 1</button>
+        <button class="btn" id="a2">Answer 2</button>
+        <button class="btn" id="a3">Answer 3</button>
+        <button class="btn" id="a4">Answer 4</button>
       </div>
     </div>
     <div class="controls">
```

Comment:
On the version I have, it was linked to script.js: `<script src="./assets/js/script.js"></script>`
Just verify that your's is linking to quiz.js: `<script src="./assets/js/quiz.js"></script>`

Reply:
I created a new js file for this since I wasn't sure if anyone else was using the script.js file. Updated the script link